### DISCUSSION TITLE
9.0.0

### DISCUSSION THREAD

**Discussion by Donnnno** [Arcticons-Team/Arcticons#2045] [Announcements]
###### Released: Mar 14, 2024
* 🎉 253 new and updated icons! (thanks @Kaiserdragon2, @fanfu, @AxFo, @willycast, @JapanYoshi, @KR4U7H3R, @Dunexus, @savetier, @Kartanovicius, @alchemiker, @TotallyAvailable, @dgcampbe, @TheLimifiedLime, @inwriting, @GaensebIume, @onuronsekiz 
* ♾️ Unlimited icon requests 4 all! We did some behind the scenes work to track icon requests better, if you're curious about the progress, [take a look here](https://arcticons-team.github.io/Arcticons/requests.html). Thanks, @Kaiserdragon2
* ✨ Updates to make sure many apps work again.
* 📁 Folder support for Discreet Launcher. Thanks to @falzonv @kaanelloed
* 🚀 A place to [check launcher compatibility](https://github.com/Arcticons-Team/Arcticons/wiki/Supported-Launchers) for all Arcticons versions.
* 🔥 9000 icons in total!

<hr /><em>This discussion was created from the release <a href='https://github.com/Arcticons-Team/Arcticons/releases/tag/9.0.0'>9.0.0</a>.</em>

**Comment by TheLimifiedLime** [comment]
Just noticed the T-Mobile Internet icon seems a bit weird. Seems it changed with commit 26a6ebdc9f95faaa68222c81a11d78ad3d284e5c. Is this intentional?

**Reply by Donnnno** [reply]
Must have been an export bug

**Reply by TheLimifiedLime** [reply]
Is it possible to fix it? I'm not sure of much of the projects structure otherwise I'd try and submit a PR.
